addons/mail/static/src/messaging/component/chatter_topbar/chatter_topbar.scss

```diff
@@ -51,12 +55,31 @@
 // Style
 // ------------------------------------------------------------------
 
-.o_ChatterTopbar .o-active {
-    color: $o-brand-odoo;
-    background-color: lighten(gray('300'), 7%);
-    border-color: gray('300');
-    border-bottom-color: transparent;
-    border-top-color: transparent;
+.o_ChatterTopbar_actions.o-has-active-button {
+    border-bottom-color: $border-color;
+}
+
+.o_ChatterTopbar_button {
+    border-radius: 0;
+
+    &:hover{
+        background-color: gray('300');
+    }
+
+    &.o-active {
+        color: $o-brand-odoo;
+        background-color: lighten(gray('300'), 7%);
+        border-left-color: $border-color;
+        border-right-color: $border-color;
+
+        &.o-bordered {
+            border-top-color: $border-color;
+        }
+
+        &:hover {
+            color: $link-hover-color;
+        }
```

Comment:
You reverted the background change on hover of active button? I think it was a good addition.

Reply:
It's only for active links, you told that it was strange that the text color wasn't the same as a non active hovered link.

Reply:
Yes it's good that it is the same text color for active, but in a previous version of this PR the background was also the same for active and I liked it, and now the background is not the same anymore for active?